routing/speed_camera_prohibition.cpp

```diff
@@ -9,7 +9,7 @@ namespace
 {
 // List of country names where mwm should be generated without speed cameras.
 std::vector<std::string> kSpeedCamerasProhibitedCountries = {
-    "Cyprus", "Macedonia", "Switzerland", "Turkey",
```

Comment:
Turkey, Switzerland, Cyprus, Macedonia

In these countries all Speed Camera warning systems are illegal. Because of this, TomTom does not offer the Speed Camera service in these countries, and users driving in these countries do not need to manually change the settings of their device.

https://help.tomtom.com/hc/en-us/articles/360013899980-TomTom-Speed-Camera-service-and-the-law